draft-ietf-cose-rfc8152bis-algs.xml

```diff
@@ -1677,23 +1682,33 @@ COSE_KDF_Context = [
         <name>Assignments for Existing Algorithms</name>
         <t>
           For the current set of algorithms in the registry, those in this document as well as those in <xref target="RFC8230"/> and <xref target="I-D.ietf-cose-hash-sig"/>, the capabilities is set to the single entry of the key type that will be accepted.
-          It is expected other algorithms will have no items or multiple items.
+          It is expected other algorithms could have zero, one, multiple items.
         </t>
 
-        <!--
-        <table>
-        <thead>
-          <tr>
-            <th>Name</th>
-            <th>Value</th>
-            <th>Capabilities</th>
-          </tr>
-          <tr>
-            
-          </tr>
-        </thead>
-        </table>
-        -->
+      </section>
+
+      <section>
+        <name>Examples</name>
+        <t>
+          In this section a trio of examples are provided.
+          In all three cases the pair of capabilities is always provided as the algoirthm and then the key capabilities.
```

Comment:
Please specify that the example uses CBOR sequence to format the "pair of capabilities", or format it in a different way (array).

s/algoirthm/algorithm

Reply:
I hate adding the reference just for this.